src/labs/utils/validate-lab.ts

```diff
@@ -31,6 +33,10 @@ export function validateLabRequest(lab: Partial<Lab>): LabError {
     labError.type = 'labs.requests.error.typeRequired'
   }
 
+  if (!lab.visitId) {
+    labError.visit = 'labs.requests.error.visitRequired'
+  }
+
```

Comment:
a test should be added to cover this piece of functionality

Reply:
I think the `validate-lab.test.ts` already cover this right?